src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.scss

```diff
@@ -1,10 +1,20 @@
+@import 'src/styles/common';
+
 .proteinSummary {
   margin-left: 178px;
   width: 695px;
 
   > div {
     padding: 10px 0 10px 0;
   }
+
+  .keyline {
+    border-bottom: 1px solid #ff9900;
```

Comment:
We have this colour in the list of variables:

<img src="https://user-images.githubusercontent.com/6834224/92147022-75b77200-ee12-11ea-8769-a302ef0a7aeb.png" width="200" />

Reply:
Oh cool. Thanks. Will change it.